backend/app/Http/Controllers/CompanyBrowserController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Inertia\Response;
+use Internships\Enums\CompanyStatus;
+use Internships\Http\Resources\DepartmentResource;
+use Internships\Models\Company;
+use Internships\Models\Department;
+
+class CompanyBrowserController extends Controller
+{
+    public function index(Request $request): Response
+    {
+        $companies = Company::where("status", CompanyStatus::Verified)->paginate(15);
+        $cities = [
+            [
+                "id" => 0,
+                "label" => "Legnica",
+            ],
+            [
+                "id" => 1,
+                "label" => "Bogatynia",
+            ],
+        ];
```

Comment:
Why we have here fixed data?

Reply:
I've written explanation in the PR description. I wanted to implement it later to push inertia into main, but maybe I'll manage to replace it in a moment.

Reply:
It now reads data from Companies.